### PR TITLE
Login form styling

### PR DESCRIPTION
**Pull request by aqadir-cc**
See commit notes.

### CODE REVIEW

**Comment by aqadir-cc** [comment]
Login form alt-style updates per comments in PT by @dpdawson

File: app/javascript/style/_authentication.scss [hunk 1]
```diff
@@ -1,7 +1,152 @@
+.logoContainer{
```
Comment (mckinley) [inline app/javascript/style/_authentication.scss]: class names are dasherized: logo-container

File: app/javascript/style/_authentication.scss [hunk 2]
```diff
@@ -1,7 +1,152 @@
...
 .authentication {
+  border-radius: 4px;
+  .checkbox-field{
+    display: inline-block;
+    width:100%;
+  }
   @extend .modal;
+  background-color: rgba(255,255,255,0.3);
+  padding:20px;
+  //border-color: $blue;
```
Comment (mckinley) [inline app/javascript/style/_authentication.scss]: Please avoid leaving commented code in PRs unless there is a good reason

File: app/javascript/style/_authentication.scss [hunk 3]
```diff
@@ -1,7 +1,41 @@
...
+      }
+    }
   }
 }
+
+.devise-sessions-new,
+.devise-passwords-create,
+.devise-passwords-new {
+  background: #1e439f;
+  background: -moz-radial-gradient(center, ellipse cover, #1e439f 0%, #00237a 69%, #00237a 100%);
```
Comment (mckinley) [inline app/javascript/style/_authentication.scss]: why has this not been fixed?

File: app/javascript/style/_masthead.scss [hunk 4]
```diff
@@ -47,3 +47,10 @@
     top: 46px;
   }
 }
+
+
+.devise-sessions-new .identity,
```
Comment (mckinley) [inline app/javascript/style/_masthead.scss]: Why is this in _masthead?

File: app/views/devise/sessions/new.html.erb [hunk 5]
```diff
@@ -1,26 +1,31 @@
...
         <%= f.label :password %>
-        <%= f.password_field :password, autocomplete: "off" %>
+        <%= f.password_field :password, autocomplete: "off", placeholder:'Enter password' %>
       </div>
 
       <% if devise_mapping.rememberable? -%>
           <div class="field checkbox-field">
             <%= f.check_box :remember_me %>
-            <%= f.label :remember_me %>
+            <%= f.label :remember_me, 'Remember Me<span></span>'.html_safe %>
```
Comment (mckinley) [inline app/views/devise/sessions/new.html.erb]: why is there an empty span here?

File: app/javascript/style/_authentication.scss [hunk 6]
```diff
@@ -1,7 +1,152 @@
...
+  button, input[type="submit"] {
+    line-height: 2;
+    vertical-align: middle;
+    padding: 8px 30px;
+    width: auto;
+    color: #fff;
+    border-radius: 4px;
+    display: block;
+    text-align: center;
+    border: 0px solid #1e439f;
```
Comment (mckinley) [inline app/javascript/style/_authentication.scss]: you've defined this as a variable and used it several times without the variable

File: app/javascript/style/_authentication.scss [hunk 7]
```diff
@@ -1,7 +1,152 @@
+.logoContainer{
+  display: block;
+  text-align: center;
+  margin-bottom: 15px;
+}
 .authentication {
+  border-radius: 4px;
+  .checkbox-field{
```
Comment (mckinley) [inline app/javascript/style/_authentication.scss]: can we please be consistent with code style. spaces before `{` and after `:`

File: app/javascript/style/_authentication.scss (hunk 6 above)
Comment (aqadir-cc) [inline app/javascript/style/_authentication.scss]: Fixed

**Comment by aqadir-cc** [comment]
@mckinley I have pushed the changes.

File: app/javascript/style/_authentication.scss (hunk 3 above)
Comment (aqadir-cc) [inline app/javascript/style/_authentication.scss]: We will be replacing it with the bg-image so I've removed gradient altogether.

File: app/javascript/style/_authentication.scss (hunk 2 above)
Comment (aqadir-cc) [inline app/javascript/style/_authentication.scss]: Noted and fixed.

File: app/javascript/style/_authentication.scss (hunk 7 above)
Comment (aqadir-cc) [inline app/javascript/style/_authentication.scss]: Code formatting fixed.

File: app/javascript/style/_authentication.scss (hunk 1 above)
Comment (aqadir-cc) [inline app/javascript/style/_authentication.scss]: Fixed and noted.

File: .gitignore [hunk 8]
```diff
@@ -13,3 +13,5 @@
 /tmp/*
 /public/packs
 /node_modules
+/.idea
```
Comment (aqadir-cc) [inline .gitignore]: Noted and fixed.

**Pull request merged**